packages/web3torrent/src/components/torrent-info/download-info/DownloadInfo.stories.tsx

```diff
@@ -41,23 +42,24 @@ storiesOf('Web3Torrent', module)
           downloadSpeed: number('Download speed (bytes/s)', 200000, {}, 'Torrent data'),
           uploadSpeed: number('Upload speed (bytes/s)', 100000, {}, 'Torrent data'),
           numPeers: number('Number of peers', 50, {}, 'Torrent data'),
+          _peers: createMockTorrentPeers(),
           wires: [
             {
               paidStreamingExtension: {
                 peerAccount: d,
-                peerChannelId: c1
+                pseChannelId: c1
               },
-              uploaded: 666
+              downloaded: 6666
             },
             {
               paidStreamingExtension: {
-                peerAccount: a,
-                peerChannelId: c2
+                peerAccount: b,
+                pseChannelId: c2
               },
-              uploaded: 333
+              downloaded: 333
             }
           ]
-        } as Torrent
+        } as unknown) as Torrent
```

Comment:
do you know why `as unknown` is needed?

Reply:
because _peers is a private, non-typed property, and I didn't want to change the types in order to make storybook work